src/nvim/eval.c

```diff
@@ -13047,6 +13051,55 @@ static void f_msgpackparse(typval_T *argvars, typval_T *rettv)
   return;
 }
 
+/// "msgpane()" function
+///
+/// Display a message in the message pane.
+///
+/// @param message Message to display.
+/// @param [highlight] The highlight group name.
+/// @param [open] Opens the message page if it's not already in a window.
+///
+/// @return Nothing.
+static void f_msgpane(typval_T *argvars, typval_T *rettv)
+{
+  char_u *msg = get_tv_string(&argvars[0]);
+  int attr = 0;
+
+  if (argvars[1].v_type != VAR_UNKNOWN) {
+    char_u *attrName = get_tv_string(&argvars[1]);
+    int hl_id = syn_name2id(attrName);
+    attr = syn_id2attr(hl_id);
+  }
+
+  if (argvars[2].v_type != VAR_UNKNOWN) {
```

Comment:
`argvars[2]` cannot always be checked. If only param `message` is used, we can only check up to `argvars[1]` and `argvars[1].v_type == VAR_UNKNOWN`.